docs/DB_API.md

```diff
@@ -626,7 +619,7 @@ Parameters:
     * uid
     * createdAt
 
-## .sessionWithDevice(tokenId) ##
+## .sessionToken(tokenId) ##
```

Comment:
Could we delete this section entirely if we added this bit and the associated `d.*` fields to the preceding section:

```
`d.*` for a field from `devices` and
```

?